src/python/dxpy/cli/output_utils.py

```diff
@@ -0,0 +1,163 @@
+import sys
+import csv
+import os
+from ..exceptions import err_exit
+
+
+def write_expression_output(
+    arg_output,
+    arg_delim,
+    arg_sql,
+    output_listdict_or_string,
+    save_uncommon_delim_to_txt=True,
+    output_file_name=None,
+):
+    """
+    arg_output: str
+    A string representing the output file path.
+    When it's "-", output is written to stdout.
```

Comment:
minor: why not "None"?

Reply:
If it's `None`, it would be written to a default <record_name>.<suffix> file.